test/unit/lookup_key_test.rb

```diff
@@ -308,6 +308,16 @@ def test_parameters_multiple_paths
     assert_include key.errors.keys, :default_value
   end
 
+  test "safe_value can be shown for key" do
+    key = FactoryGirl.build(:puppetclass_lookup_key, :as_smart_class_param, :hidden_value => false,
+                            :override => true, :key_type => 'string',
+                            :default_value => 'aaa', :puppetclass => puppetclasses(:one))
+    assert key.safe_value
+    key.hidden_value = true
+    key.save!
+    assert key.safe_value
```

Comment:
Can you change both `assert` checks to `assert_equal`?  This is only checking if there's an error right now, not that it's doing the right thing.